wordpress-mobile/WordPress-iOS
# Subscribers List Details: Update People List UI

Fixes #23046

- Update the People List UI so it can be used when navigating from the Subscribers card.
- Navigate to WP.com/subscribers page when a subscriber is selected.
- Remove Email followers tab.

Figma designs: IqhXWz3Iir7RMb5XH5gGfZ-fi-1137_3593

## To test:

1. Enable `Subscribers and Traffic tab` feature flag
2. Open Stats -> Subscribers
3. On Subscriber list select "View more"
4. Confirm People -> Subscribers is opened with an updated UI
5. Tap on Subscriber
6. Confirm a web view with subscriber information is opened
7. Switch to People -> Users tab and confirm an updated UI appears

| 1 | 2 | 3 | 4 |
|--------------|--------------|--------------|--------------|
| ![Simulator Screenshot - iPhone 15 - 2024-04-29 at 17 51 59](https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/160a6950-77c5-4ec2-a92d-02d28c50b972) | ![Simulator Screenshot - iPhone 15 - 2024-04-29 at 17 50 04](https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/bbb21198-8ba1-42bb-94a9-19e12e467a42) | ![Simulator Screenshot - iPhone 15 - 2024-04-29 at 17 49 52](https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/12c556a2-0fd3-42f0-bd61-d1f3b9b0d09d) | ![Simulator Screenshot - iPhone 15 - 2024-04-29 at 17 49 33](https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/6e2a5a93-2ad3-420e-a73f-d93ff403e8a6) |

## Regression Notes
1. Potential unintended areas of impact

Existing People views, especially People -> Users would continue working as before. 

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Manual testing

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


Testing checklist:
